core/src/main/scala/org/apache/spark/api/conda/CondaEnvironment.scala

```diff
@@ -113,17 +114,33 @@ final class CondaEnvironment(
   }
 
   /**
-   * This is for sending the instructions to the executors so they can replicate the same steps.
+   * This is for sending the instructions to the executors so they can replicate the same conda
+   * environment.
+   * For `File` bootstrap mode, use original setup
+   * For `Solve` bootstrap mode, convert to `File` mode to use exactly same conda pkgs
+   * and avoid re-solving.
```

Comment:
Nit: Formatting and maybe additional documentation (please add / correct me if I'm wrong).
```suggestion
   * environment.
   * <p><ul>
   *   <li>In {@code File} mode, re-use the same environment file on executors.
   *   <li>In {@code Solve} mode, list resolved packages into an environment file and use that on executors.
   * </ul>
```